std/experimental/ndslice/slice.d

```diff
@@ -1737,17 +1738,27 @@ struct Slice(size_t _N, _Range)
             if (this._strides == rslice._strides && this._ptr == rslice._ptr)
                 return true;
         }
-        return opEqualsImpl(this, rslice);
+        foreach (i; Iota!(0, PureN))
+            if (this._lengths[i] == 0)
+                return true;
```

Comment:
@WalterBright patch coverage — 96.36%. If Phobos is crazy about 100 coverage now, please make an official announce, that NO PRs should be merged with coverage <100%! I don't want to be random victim for coverage idealists!